### DISCUSSION TITLE
Does PushMenu noTransitionAfterReload function meet expectations?

### DISCUSSION THREAD

**Discussion by ZhangChengLin** [ColorlibHQ/AdminLTE#3699] [General]
https://github.com/ColorlibHQ/AdminLTE/blob/654ad02d8ad90f98c35ead5a1d3e47fe09c3e9e6/build/js/PushMenu.js#L121-L146

**Comment by danny007in** [comment]
create this as issue

**Reply by ZhangChengLin** [reply]
I'm not sure, just ask, because I feel like nothing has changed.

**Reply by ZhangChengLin** [reply]
I'm going to sleep, and a woman won't let me stay up late anymore.
